website/docs/_posts/2017-10-05-configuration-options.md

```diff
@@ -178,15 +187,59 @@ For example, To add `[skip ci]` to every commit you could configure:
 
 Commit message template
 
-| name    | value                                                                                          |
-| ------- | ---------------------------------------------------------------------------------------------- |
-| type    | handlebars template                                                                            |
-| default | {% raw %}{{semanticPrefix}}Update dependency {{depName}} to version {{newVersion}}{% endraw %} |
+| name    | value                 |
+| ------- | --------------------- |
+| type    | handlebars template   |
+| default | {% raw %}{% endraw %} |
 
 The commit message is less important than branchName so you may override it if you wish.
 
 Example commit message: "chore(deps): Update dependency eslint to version 4.0.1"
 
+## commitMessageAction
+
+Action verb to use in commit messages and PR titles.
+
+| name    | value  |
+| ------- | ------ |
+| type    | string |
+| default | Update |
+
+This is used to alter `commitMessage` and `prTitle` without needing to copy/paste the whole string. Actions may be like 'Update', 'Pin', 'Roll back', 'Refresh', etc.
+
+## commitMessageExtra
+
+Extra description used after the commit message topic - typically the version.
+
+| name    | value  |
+| ------- | ------ |
+| type    | string |
+| default | Update |
```

Comment:
Is the `default` actually `Update`?

Reply:
yes

Reply:
I thought `commitMessageAction` was `Update`, while `commitMessageExtra` was `to 16.0.0`, or whatever?

Reply:
Yes, that's how it is in the final doc